.github/workflows/test.yml

```diff
@@ -17,7 +17,7 @@ jobs:
       - uses: actions/checkout@v3
       - uses: actions/setup-node@v3
         with:
-          node-version: '14'
+          node-version: '16'
```

Comment:
Issue：Thanks.
Using node v 14 in yamory, update it too.
https://github.com/garoon/plugin-uploader/blob/main/.github/workflows/yamory.yml

Reply:
Thanks. I updated it.